cmd/subctl/diagnose.go

```diff
@@ -210,7 +210,7 @@ func clusterInfoFromKubeConfig(kubeConfig string) *cluster.Info {
 	clientProducer, err := client.NewProducerFromRestConfig(config.Config)
 	exit.OnErrorWithMessage(err, fmt.Sprintf("Error creating client producer for kubeconfig %q", kubeConfig))
 
-	clusterInfo, err := cluster.NewInfo("", clientProducer)
+	clusterInfo, err := cluster.NewInfo("", clientProducer, nil)
```

Comment:
This is unfortunate, but let’s not block the PR — see #2042.

Reply:
Agree.